CHANGELOG.md

```diff
@@ -6,8 +6,11 @@
 
 ## Breaking Changes
 
+- [#1916](https://github.com/oauth2-proxy/oauth2-proxy/pull/1916) Keycloak OIDC provider was using access token instead of id token. After upgrade, it may raise errors on login for people using provider "keycloak-oidc" (not "keycloak") and having customized their tokens in keycloak configuration.
```

Comment:
Yes, I'm still here! Yes, I'm still motivated.
I'll make some tests, but I think that people who have already customized configuration should know how to remove it. And if they only add new fields, the change should not affect them since they do not change the existing fields.
But I can make it more clear if you think it can be an issue!

Reply:
@Elektordi @kvanzuijlen any news on this topic ?